service/src/test/java/uk/gov/hmcts/reform/fpl/config/GeneralInboxLookupConfigurationTest.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class GeneralInboxLookupConfigurationTest {
+    private static final String GENERAL_INBOX = "FamilyPublicLaw+generalInbox@gmail.com";
+
+    private GeneralInboxLookupConfiguration generalInboxLookupConfiguration
+        = new GeneralInboxLookupConfiguration(GENERAL_INBOX);
+
+    @Test
+    void shouldReturnGeneralInbox() {
```

Comment:
Is it really necessary to assert that a constructor works?

Reply:
as discussed keeping it as we are testing for other configuration as well.

Reply:
Still don't like...

Reply:
removed